docs/features/migrate_to_pootle_fs.rst

```diff
@@ -79,13 +79,36 @@ plugin.  For examples for Git:
 This is done once for the whole Pootle server.
 
 
+.. _migrate_to_pootle_fs#connect-with-vcs:
```

Comment:
Same problem with this move.  Migration is once off and now setup instructions are hidden here.  The correct approach is the other way around.  Migration, once off, should point to general setup instructions as needed.